app/Http/Controllers/Api/AssetsController.php

```diff
@@ -49,10 +51,32 @@ class AssetsController extends Controller
      * @since [v4.0]
      * @return JsonResponse
      */
-    public function index(Request $request, $audit = null)
+    public function index(Request $request, $audit = null) 
     {
 
-        $this->authorize('index', Asset::class);
+        \Log::debug(Route::currentRouteName());
```

Comment:
This is debatable if we want to keep this, seems a little inside-baseball.

Reply:
You're 100% right - that was just for debugging and should be removed. I'll do that.

Reply:
Actually, nvm - that's not even there anymore AFAIK